test/integration/default/upload-notebook-test.js

```diff
@@ -125,7 +123,7 @@ describe('upload notebook', function() {
         });
     });
 
-    it('should upload a notebook to a subdirectory', function() {
+    it('should upload a notebook to a subdirectory', function(done) {
```

Comment:
Not sure how it slipped through, but of course this isn't run in Travis since its integration.